package.json

```diff
@@ -9,6 +9,8 @@
     "babel-preset-es2015": "^6.24.1",
     "browser-sync": "2.x",
     "cheerio": "^1.0.0-rc.2",
+    "eslint": "2.x",
```

Comment:
I think we can bump this back to `4.x`.  We downgraded to 2.x to debug and forgot to put it back.